src/Conventional_Monitor/image_gen/ush/plot_time_ps.sh

```diff
@@ -29,9 +24,8 @@ set -ax
    #  plot surface pressure time series counts
    #---------------------------------------------------
 
-   cp -f ${C_IG_GSCRIPTS}/plotstas_time_count_ps.gs . 
-   cp -f ${C_IG_GSCRIPTS}/plotstas_time_bias_ps.gs  . 
-#   cp -f ${C_IG_GSCRIPTS}/plotstas_time_bias2_ps.gs . 
+   ${NCP} ${C_IG_GSCRIPTS}/plotstas_time_count_ps.gs . 
+   ${NCP} ${C_IG_GSCRIPTS}/plotstas_time_bias_ps.gs  . 
```

Comment:
plotstas_time_bias_ps.gs does not exist.  The script plotstas_time_bias2_ps.gs does, however.

Reply:
Replaced `plotstas_time_bias_ps.gs` with `plotstas_time_bias2_ps.gs` and confirmed it runs without error and produces expected output.